.travis.yml

```diff
@@ -1,3 +1,5 @@
+if: NOT (commit_message =~ /^\[docs\]/) OR (type = push AND branch = master) OR (type = pull_request AND NOT branch = master)
```

Comment:
```
if: NOT (commit_message =~ /^\[docs\]/) OR (branch = master) OR (type != push)
```
This way is simpler, and enables restarting tests via Travis API (required for web interface and build bot).

Reply:
Ok